src/components/application_manager/src/mobile_message_handler.cc

```diff
@@ -232,7 +232,7 @@ MobileMessageHandler::HandleOutgoingMessageProtocolV2(
 
   const size_t dataForSendingSize =
       protocol_handler::PROTOCOL_HEADER_V2_SIZE + jsonSize + binarySize;
-  uint8_t* dataForSending = new uint8_t[dataForSendingSize];
+  std::vector<uint8_t> dataForSending(dataForSendingSize);
```

Comment:
@vlantonov We have a typedef for this type of data, please use it. I assume it called BinaryData